radio/src/targets/horus/hal.h

```diff
@@ -701,6 +701,8 @@
   #define TOUCH_RST_GPIO                  GPIOF
 #if defined(PCBX12S)
   #define TOUCH_RST_GPIO_PIN              GPIO_Pin_7   // PF.7
+#if defined(RADIO_T16)
+  #define TOUCH_RST_GPIO_PIN              GPIO_Pin_10   // PF.10
```

Comment:
You do not need to define this explicitly for T16, as PF10 would be handled by the #else case already anyhow.

Reply:
only from tests...